src/app/pages/StoryPage/StoryPage.jsx

```diff
@@ -238,6 +238,71 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
               />
             );
           }
+          if (variation === 'variation_1') {
```

Comment:
LGTM.
It appears that the difference in implementation across the different variations,
is the attribute from which we fetch recommendations in pageData.
Please consider using a mapping object to retrieve recommendations data.
See example below:
```
const mapping = {
    "content_recs": "datalabContentRecommendations"
    "hybrid_recs": "datalabHybridRecommendations",
    "variation_1": "datalabHybridRecommendationsV1x1",
    ...
}
const recommendationID = mapping[variation];
const unirecsHybridRecommendationData = path(
    [recommendationID],
    pageData,
);
...
```